api/nodes.go

```diff
@@ -89,11 +89,12 @@ func (n *Nodes) Stats(nodeID string, q *QueryOptions) (*HostStats, error) {
 	if err != nil {
 		return nil, err
 	}
-	var hostStats HostStats
-	if _, err := client.query("/v1/client/stats/", &hostStats, nil); err != nil {
+	var resp []HostStats
```

Comment:
The api should let you specify the timestamp since. For this and allocations stats